xml/art_solution_automation.xml

```diff
@@ -0,0 +1,438 @@
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&terraform;</term>
+        <listitem>
+          <remark>toms 2020-06-10: https://www.terraform.io/intro/index.html</remark>
+          <para>
+            An <quote>infrastructure as code</quote> software tool.
+            It deploys the needed infrastructure in the cloud or virtual environments
+            and &ay; for on-premises deployments.
+          </para>
+          <para>
+            Currently, &terraform; is <emphasis role="strong">NOT</emphasis> supported
+            by &suse;.
+          </para>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-using-salt-formulas">
+    <title>Using &salt; Formulas</title>
+    <remark>toms 2020-06-29: Is there a general way to install/use these? Should
+    we tell our readers about that?</remark>
+    <para>
+      To use &salt; formulas from &suse;, execute the following steps:
+    </para>
+    <!--
+    <procedure>
+      <step>
+        <para>Install the </para>
+        <screen>git clone https://github.com/SUSE/shaptools.git
+cd shaptools
+sudo python setup.py install</screen>
+      </step>
+      <step>
+        <para></para>
+      </step>
+    </procedure>
+    -->
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-hana-deploy">
+    <title>&hana; Automated Deployment</title>
+    <para>
+      &salt; formula <package>saphanabootstrap-formula</package> for bootstrapping
+      and managing &hana; platform and system replication.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>See <link xlink:href="https://github.com/SUSE/saphanabootstrap-formula"/>.</para>
+    </formalpara>
+    <itemizedlist>
+      <title>Capabilities for <package>saphanabootstrap-formula</package></title>
+      <listitem>
+        <para>
+          Installs &hana;.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures system replication.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures &ha; cluster.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Configures optional monitor exporter.
+        </para>
+      </listitem>
+    </itemizedlist>
+    <para>
+      The &salt; formula can be used in these scenarios:
+    </para>
+    <itemizedlist>
+      <title>Scenarios for <package>saphanabootstrap-formula</package></title>
+      <listitem>
+        <para>
+          &hana; single instance automated deployment.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Two &hana; instances on system replication setup.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Two &hana; instances on system replication and cluster scale-up
+          performance-optimized configuration.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
```

Comment:
I don't know if I would put this capability here. The cluster setup (performance or cost-optimized) is something that belongs to the HA cluster and not the HANA itself. But it is in a grey area anyway... It's hard to decide where this information should go.

Reply:
I will discuss that with Tanja.